scripts/india_nhm/.gitignore

```diff
@@ -0,0 +1,2 @@
+# pylint directory
```

Comment:
remove all the .pylint.d files from the PR?

Reply:
Done.